modules/op-release-notes-1-8.adoc

```diff
@@ -50,8 +314,147 @@ Workaround: Use `nodejs:14-ubi8` instead of the `nodejs:14-ubi8-minimal` image s
+// link:https://github.com/tektoncd/pipeline/issues/5095
+// Vincent Demeester @vdeemester
+
+* Before this update, the controller might generate multiple panics if pulling an image failed, or its pull status was incomplete. This update fixes the issue by checking the `step.ImageID` value instead of the `status.TaskSpec` value.
+// link:https://github.com/tektoncd/pipeline/pull/5245(#5245)
+// Piyush Garg @piyush-garg
+
+* Before this update, canceling a pipeline run that contained an unscheduled custom task produced a `PipelineRunCouldntCancel` error. This update fixes the issue. You can cancel a pipeline run that contains an unscheduled custom task without producing that error.
+// link:https://github.com/tektoncd/pipeline/issues/5295(#5295)
+//Andrew Bayer @abayer
+
+
+* Before this update, if the `<NAME>` in `$params["<NAME>"]` or `$params['<NAME>']` contained a dot character (`.`),  any part of the name to the right of the dot was not extracted. For example, from `$params["org.ipsum.lorem"]`, only `org` was extracted.
++
+This update fixes the issue so that `$params` fetches the complete value. It also throws an error if `<NAME>` is not enclosed in single or double quotes. For example:
++
+* `$params.org.ipsum.lorem` is not valid and generates a validation error.
+* `$params["org.ipsum.lorem"]` and `$params['org.ipsum.lorem']` are valid and the entire value of `<NAME>`, `org.ipsum.lorem`, is extracted.
+// link:https://github.com/tektoncd/pipeline/pull/4880
+
+// .Triggers
+
+With this update, `Trigger` resources support custom interceptors and ensure that the port of the custom interceptor service is the same as the port in the `ClusterInterceptor` definition file.
+// link:https://github.com/tektoncd/triggers/pull/1383[(#1383)]
+// @savitaashture
+
+
+// .CLI
+
+* Before this update, the `tkn version` command for {tekton-chains} and Operator components didn't work correctly. This update fixes the issue so that the command works correctly and returns version information for those components.
+// https://github.com/tektoncd/cli/pull/1542
+// Piyush Garg @piyush-garg
+
+* Before this update, if you ran a `tkn pr delete --ignore-running` command and a pipeline run did not have a `status.condition` value, the `tkn` CLI tool produced a null-pointer error (NPE). This update fixes the issue so that the CLI tool now generates an error and correctly ignores pipeline runs that are still running.
+// https://github.com/tektoncd/cli/pull/1532
+// PRADEEP KUMAR @pradeepitm12
+
+* Before this update, if you used the `tkn pr delete --keep <value>` or `tkn tr delete  --keep <value>` commands, and the number of pipeline runs or task runs was less than the value, the command did not return an error as it should. This update fixes the issue so that the command correctly returns an error under those conditions.
```

Comment:
```suggestion
* Before this update, if you used the `tkn pr delete --keep <value>` or `tkn tr delete  --keep <value>` commands, and the number of pipeline runs or task runs was less than the value, the command did not return an error as expected. This update fixes the issue so that the command correctly returns an error under those conditions.
```

Reply:
Updated. Thanks.